packages/flutter/test/cupertino/dialog_test.dart

```diff
@@ -331,12 +269,389 @@ void main() {
     // Check that there's no button action section.
     expect(scrollController.offset, 0.0);
     expect(find.widgetWithText(CupertinoDialogAction, 'One'), findsNothing);
+
+    // Check that the dialog size is the same as the content section size. This
+    // ensures that an empty button section doesn't accidentally render some
+    // empty space in the dialog.
+    expect(
+      tester.getSize(find.byKey(const Key('cupertino_alert_dialog_content_section'))),
+      tester.getSize(find.byKey(const Key('cupertino_alert_dialog_modal'))),
+    );
+  });
+
+  testWidgets('Actions section height for 1 button is height of button.',
+          (WidgetTester tester) async {
```

Comment:
nit: some indentation issues here and below

Reply:
what should the indentation be?

Reply:
```dart
testWidgets(
  'blah',
  (WidgetTester tester) async {
    foo();
  },
);
```
or keep it in one line. 

Though if dartfmt did this, I don't have a strong opinion either way.

Reply:
Do you know of other tests that are formatted the way you describe with the test name on the 2nd line?  Off the top of my head, I don't think I've seen that.  I mean I've seen it for widgets a lot, but not for tests.

Reply:
Search for `testWidgets\($` in the codebase

Reply:
technically what you have here also matches our style guide (though maybe the second line is a bit overindented).
Personally though I'd just put it all on one line. We have wide monitors. :-)